angular-with-directus/how to integrate directus with Angular.md

```diff
@@ -0,0 +1,396 @@
+
+To ensure the collection is a singleton, select the **Singleton** checkbox. Note that this collection matches the `Metadata` type you created in the previous section.
+
+Once the collection is defined go to the **Content** section and add the title and description for the metadata. 
+
+To permit public access to the metadata, go to **Settings > Access Control > Public** and permit the read permission under the metadata section.
+
+### Create a Component for the Metadata
+Use the following command on a terminal window pointing to your project.
+
+```bash
+ng g c component/home
+```
+This command will generate four files under the *component* directory. 
+
+Replace the code in the `src/app/component/home/home.component.ts` file with the following code.
+
+```ts
+import {Component, OnInit} from '@angular/core';
+import {directus, Metadata} from "../../../../directus";
+import {CommonModule} from "@angular/common";
+import {readSingleton} from "@directus/sdk";
+
+@Component({
+  selector: 'app-home',
+  standalone: true,
+  imports: [
+    CommonModule
+  ],
+  templateUrl: './home.component.html',
+  styleUrl: './home.component.css'
+})
+export class HomeComponent implements OnInit{
+  metadata: Metadata;
+  ngOnInit(): void {
+    this.getMetadata();
+  }
+  async getMetadata(){
+    //@ts-ignore
+    this.metadata = await directus
+      .request<Metadata>(readSingleton("metadata"))
+  }
+}
+```
+When this component is initialized, it will retrieve the singleton and store it in the `metadata` object. 
+
+To display the contents of the object, replace the code in the `src/app/component/home/home.component.html` file with the following code.
+
+```ts
+<div *ngIf="metadata">
+  <h1>{{metadata.title}}</h1>
+  <p>{{metadata.description}}</p>
+</div>
+```
+### Add Routing for the Metadata
+In your project, go to `app.routes.ts` and replace the code in the file with the following code.
+
+```ts
+import { Routes } from '@angular/router';
+import {HomeComponent} from "./component/home/home.component";
+
+export const routes: Routes = [
+  {path: '', component: HomeComponent},
+];
+```
+Once the application reloads, go to `https://localhost:4200`. As a result, the home component containing the metadata is loaded on the page.
+
+## Serving Dynamic Pages
+In your Directus project, create a new collection named `job` with the fields `id`, `title` , and `description`.
+
+However, the collection should not be a singleton. In addition, grant public access and add some job items to the collection. 
+
+Angular uses path variables and query parameters to serve dynamic data. As a result, you will leverage the `id` as the path variable to display different type of job items.
+
+### Create a Component for Dynamic Pages
+On an existing terminal window of your project, use the following command to generate the the job detail component.
```

Comment:
Ah interesting - so you always need a second terminal. I would amend to this: 

```suggestion
In your terminal, generate the the job detail component:
```

Of course, I want you to use the correct collection names, but I would still simplify language for clarity.